airflow_provider_tm1/hooks/tm1.py

```diff
@@ -46,9 +55,8 @@ def __init__(
         self.password = conn.get_password()
 
         # get relevant extra params
-        extras = conn.extra_dejson
-        self.ssl: bool = extras.get("ssl", False)
-        self.session_context: str = extras.get("session_context", "Airflow")
+        self.ssl: bool = extra["ssl"] == "True"
```

Comment:
@Cubewise-JoeCHK do we need to make `extra["ssl"]` mandatory? Cannot we use 
`extra.get("ssl", "False") == "True" `
or even
`extra.get("ssl") == "True"`
instead?

Reply:
in normal TM1py use case, SSL parameter is not a must. 

the user may directly input as a single parameter "https://localhost:8099".
instead of localhost, 8099, ssl=True
(both should be equivalent, @MariusWirtz should we restrict the input format to make it consistent?)

`extra.get("ssl", "False") == "True"` I didn't consider data type of extra['ssl'] before. I agree with this proposal. 

for the future, it should be better to have a dropdown or checkbox to control the ssl field.